tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,centos7-mon-osd,centos7-cluster}
+envlist = {ansible2.2}-{xenial-conf-tests,xenial-mon-osd,xenial-cluster,raw-multi-journal,centos7-cluster,dmcrypt-dedicated-journal,dmcrypt-journal-collocation}
```

Comment:
I've named them this way so they match up with the name of the actual config setting, but I can change them if you feel strongly. If you want to assume a dedicated journal on these tests then we might want to change ``xenial-cluster`` and ``centos7-cluster`` to use a dedicated journal and change the ``raw-multi-journal`` scenario to ``journal-collocated``.